src/less/general.less

```diff
@@ -1,5 +1,13 @@
-.mw-parser-output .editor-token.active {
-	background-color: #facc48;
+.mw-parser-output .editor-token {
+	cursor: pointer;
+
+	a {
```

Comment:
We actually might want to do this for all links when the system's active... is that covering them all or just tokenized ones?

This is probably not for this task, but I wonder if we can (or should?) disable the hovercards behavior. Should we? I'll ping Ilana for that question.

Reply:
The `pointer-events: none` will disable hovercards, but I don't think they'd work anyway because WWT replaces the content and the hovercard listeners aren't re-added.

This does only target tokenized links. I do think any non-tokenized area (like infoboxes) should be completely un-clickable, and possibly grayed out (as the greasemonkey version does).